Solutions/GitHub/Data Connectors/GithubWebhook/GithubWebhook_API_FunctionApp.json

```diff
@@ -0,0 +1,133 @@
+					"action": true
+				}
+			}
+		],
+		"customs": [{
+				"name": "Microsoft.Web/sites permissions",
+				"description": "Read and write permissions to Azure Functions to create a Function App is required. [See the documentation to learn more about Azure Functions](https://docs.microsoft.com/azure/azure-functions/)."
+			}
+		]
+	},
+	"instructionSteps": [{
+			"title": "",
+			"description": ">**NOTE:** This connector uses Azure Functions which is a http trigger based function app. It will be configured as github webhook and gets the events into Microsoft Sentinel. This might result in additional costs for data ingestions. Check the [Azure Functions pricing page](https://azure.microsoft.com/pricing/details/functions/)."
+		},
+		{
+			"title": "",
+			"description": ">**(Optional Step)** Securely store workspace and API authorization key(s) or token(s) in Azure Key Vault. Azure Key Vault provides a secure mechanism to store and retrieve key values. [Follow these instructions](https://docs.microsoft.com/azure/app-service/app-service-key-vault-references) to use Azure Key Vault with an Azure Function App."
+		},
+		{
+			"description": ">**NOTE:** This data connector depends on a parser based on a Kusto Function to work as expected [**GitHub**](https://aka.ms/sentinel-GitHub-parser) which is deployed with the Microsoft Sentinel Solution."
+		},
+		{
+			"title": "",
+			"description": "**STEP 1 - Configuration steps for the GitHub webhook API**"
+		},
+		{
+			"title": "",
+			"description": "**STEP 2 - Choose ONE from the following two deployment options to deploy the connector and the associated Azure Function**\n\n>**IMPORTANT:** Before deploying the GitHub data connector, have the Workspace ID and Workspace Primary Key (can be copied from the following).",
+			"instructions": [{
+					"parameters": {
+						"fillWith": [
+							"WorkspaceId"
+						],
+						"label": "Workspace ID"
+					},
+					"type": "CopyableLabel"
+				},
+				{
+					"parameters": {
+						"fillWith": [
+							"PrimaryKey"
+						],
+						"label": "Primary Key"
+					},
+					"type": "CopyableLabel"
+				}
+			]
+		},
+		{
+			"title": "",
+			"description": "**Option 1 - Azure Resource Manager (ARM) Template**\n\nUse this method for automated deployment of the GitHub data connector using an ARM Tempate.\n\n1. Click the **Deploy to Azure** button below. \n\n\t[![Deploy To Azure](https://aka.ms/deploytoazurebutton)](https://aka.ms/sentinel-GitHubwebhookAPI-azuredeploy)\n2. Select the preferred **Subscription**, **Resource Group** and **Location**. \n> **NOTE:** Within the same resource group, you can't mix Windows and Linux apps in the same region. Select existing resource group without Windows apps in it or create new resource group.\n3. Enter the **GitHubEnvId**, **GitHubAppName**, **GitHubUsername** and **GitHubPassword**  and deploy. \n4. Mark the checkbox labeled **I agree to the terms and conditions stated above**. \n5. Click **Purchase** to deploy."
```

Comment:
This is not matching with what I experienced. Can you please update the steps which are actually there.

Reply:
corrected the content